kml/serdes_gpx.cpp

```diff
@@ -27,9 +27,12 @@ std::string_view constexpr kWpt = "wpt";
 std::string_view constexpr kRtePt = "rtept";
 std::string_view constexpr kName = "name";
 std::string_view constexpr kColor = "color";
+std::string_view constexpr kOsmandColor = "osmand:color";
```

Comment:
Looks like `using namespace std::string_view_literals;` above is not used in the file.